source/util.js

```diff
@@ -23,3 +28,21 @@ export async function isNotificationTargetPage(url) {
 	// Issue, PR, commit paths, and per-repo notifications
 	return /^(((issues|pull)\/\d+(\/(commits|files))?)|(commit\/.*)|(notifications$))/.test(repoPath);
 }
+
+export function parseLinkHeader(header) {
+	return header.split(',').reduce((links, part) => {
+		const [sectionUrl, sectionName] = part.split(';');
```

Comment:
```suggestion
		const [sectionUrl = '', sectionName = ''] = part.split(';');
```

Same reason as [above](https://github.com/sindresorhus/notifier-for-github/pull/192#discussion_r331778082).